x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -199,8 +199,12 @@ bulk request on an index or data stream that contains new fields that may
 be mapped rather than an explicit <<indices-put-mapping,update mapping>> request.
 
 `create`::
-Privilege to index documents. Also grants access to the update mapping
-action.
+Privilege to index documents.
++
+deprecated:[8.2] Also grants the permission to update the index mapping (but not
+the data stream mapping), using the update mapping action, or relying on
+{ref}/dynamic-mapping.html[dynamic mappings]. In the next major release,
```

Comment:
Should we use a more generic statement like "in a next major release" instead? Explicitly mentioning the next major release already caused some problems in the past, and I'm not sure that it gives more value than a generic one. The same in all other changes.

What do you think?

Reply:
I suggested "a future major release" in my comments.

Reply:
I have accepted "a future major release".